toolkit/tools/go.mod

```diff
@@ -1,25 +1,36 @@
 module github.com/microsoft/CBL-Mariner/toolkit/tools
 
-go 1.13
+go 1.17
 
 require (
 	github.com/alecthomas/template v0.0.0-20190718012654-fb15b899a751 // indirect
 	github.com/alecthomas/units v0.0.0-20190924025748-f65c72e2690d // indirect
 	github.com/bendahl/uinput v1.4.0
 	github.com/cavaliercoder/go-cpio v0.0.0-20180626203310-925f9528c45e
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/gdamore/encoding v1.0.0 // indirect
 	github.com/gdamore/tcell v1.3.0
 	github.com/jinzhu/copier v0.3.2
 	github.com/juliangruber/go-intersect v1.1.0
 	github.com/klauspost/compress v1.10.5 // indirect
 	github.com/klauspost/pgzip v1.2.3
+	github.com/konsorten/go-windows-terminal-sequences v1.0.3 // indirect
```

Comment:
I think I've got a similar set of updates in #3016 but split into two `require` sections (one with direct and one with indirect dependencies) by running `sudo make go-tidy-all`. My version of Go is 1.17.3. I'd prefer to stick to the output from that `make` command, so we don't end up updating from one version to the other, so I'm curious, how did you get your changes. My guess is that it might be the difference in the Go versions we're using.

Reply:
I concur, instead of running `go mod tidy` and doing anything with the go.mod manually.
It makes sense to stick to the output from `make go-tidy-all`.